CHANGELOG.md

```diff
@@ -20,6 +20,36 @@ Code v99.99.999
 
 -->
 
+## [4.5.2](https://github.com/coder/code-server/releases/tag/v4.5.2) - 2022-08-15
+
+Code v1.68.1
+
+### Security
+
+- Fixed the proxy route not perfoming authentication. For example if you were
```

Comment:
Small suggestion, there's a typo in `perfoming`

Reply:
Thank you!